docs/SIDDHI_AE_SETUP.md

```diff
@@ -0,0 +1,37 @@
+# TPS Siddhi AE
```

Comment:
Add a sentence or more saying what this is.

Something like:
This is an implementation of transparent security analytics engine based on Siddhi.  It is able to identify a DDoS attack in about 1 second by examining  telemetry reports generated by a switch or router.  This will also identify when a mitigated attack has based on drop telemetry reports from the SDN controller.